packages/custom/battle/public/routes/battle.js

```diff
@@ -2,9 +2,9 @@
 
 angular.module('mean.battle').config(['$stateProvider',
   function($stateProvider) {
-    $stateProvider.state('battle example page', {
-      url: '/battle/example',
-      templateUrl: 'battle/views/index.html'
+    $stateProvider.state('problem', {
+      url: '/battle/problem',
+      templateUrl: 'battle/views/problem.html'
```

Comment:
It changed it to problem.html just so I could keep around the old one as a reference. If we want to change it back, I can do that.